### DISCUSSION TITLE
How to render a document like RStudio?

### DISCUSSION THREAD

**Discussion by youcc** [posit-dev/positron#3674] [Feedback]
How to render a document in Positron like RStudio? Will render feature be added so that R Markdown or Quarto project can be rendered?

**Comment by jmcphers** [comment]
You can render Quarto projects by installing the Quarto extension -- it works in Positron very similarly to how it works in Visual Studio code. Instructions here:

https://quarto.org/docs/tools/vscode.html

There's also a "Render Document with R Markdown" command you can use if you are looking at an `.Rmd` file in Positron. A little more context on that here: https://github.com/posit-dev/positron/pull/3308

**Reply by youcc** [reply]
Thanks for the quick reply! It works great rendering either Rmd or Quarto project after installing the extension.

**Comment by CorradoLanera** [comment]
@juliasilge , when trying to preview an md/Rmd file, Positron signals an error requiring a version of Quarto greater than 1.5.39 even though it has installed and activated the 1.114.0.

NOTE: right click + "render with rmarkdown" will render the Rmd, but trying to preview it (or the rendered md) will lead to this issue.

```
Positron Version: 2024.08.0 (system setup) build 48
Code - OSS Version: 1.91.0
Commit: ed616b36655fb24d116108bdd833f1321704315b
Date: 2024-08-19T03:30:12.447Z
Electron: 29.4.0
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Windows_NT x64 10.0.26120
```

![Screenshot 2024-08-23 004359](https://github.com/user-attachments/assets/7464a3f5-1e39-4e39-8594-601757eac424)

**Reply by juliasilge** [reply]
Can you check the version of Quarto you have? You can do that from the command line via `quarto -v` or in Positron on the bottom bar:

![Screenshot 2024-08-22 at 8 17 55 PM](https://github.com/user-attachments/assets/09819cf1-c219-41aa-a814-0e074194a054)

The version I see in your screenshot is for the Quarto _extension_, not Quarto itself; these are separate.

**Reply by CorradoLanera** [reply]
1.4.420, mmm. So 1.114.0 is the version of the Quarto extension but not of the Quarto engine. OK, makes sense; I wrongly supposed that the extension comes with the corresponding/updated version of the engine (which... does not make sense 😆 ). Thank you!
